data-prepper-plugins/s3-source/src/integrationTest/java/org/opensearch/dataprepper/plugins/source/SqsWorkerIT.java

```diff
@@ -59,12 +61,14 @@ void setUp() {
                 .build();
         bucket = System.getProperty("tests.s3source.bucket");
         s3ObjectGenerator = new S3ObjectGenerator(s3Client, bucket);
-        s3EventMessageParser = new S3EventMessageParser();
 
         sqsClient = SqsClient.builder()
                 .region(Region.of(System.getProperty("tests.s3source.region")))
                 .build();
 
+//        notificationSource = System.getProperty("tests.s3source.notification.source");
```

Comment:
Did you mean to leave all this commented out?

Reply:
No, this is supposed to be removed. Let me update it